starknet_py/tests/e2e/docs/code_examples/test_contract.py

```diff
@@ -91,7 +137,39 @@ async def test_deploy_contract(account, class_hash):
         constructor_args={"value": 1},
         max_fee=int(1e15),
     )
-    # docs-end: deploy_contract
+    # docs-end: deploy_contract_v1
+
+
+@pytest.mark.asyncio
+async def test_deploy_contract_v3(account, cairo1_hello_starknet_class_hash: int):
+    compiled_contract = load_contract("HelloStarknet")["sierra"]
+    abi = create_sierra_compiled_contract(
+        compiled_contract=compiled_contract
+    ).parsed_abi
```

Comment:
I think it would actually make sense to include this in the example. It might not be obvious how to obtain a contract's ABI.